### PR TITLE
i18(zh-cn): Synchronizing content

### PR DESCRIPTION
**Pull request by GuoJikun**
#### What kind of changes does this PR include?

- Translated content

### CODE REVIEW

File: src/content/docs/zh-cn/start/frontend/nextjs.mdx [hunk 1]
```diff
@@ -1,103 +1,118 @@
...
-    assetPrefix: isProd ? null : `http://${internalHost}:3000`,
-  };
-  return nextConfig;
-};
-```
+<Steps>
+
+1.  ##### 安装 `internal-ip` 版本 7 以进行移动端开发。
+
+    版本 8.0.0 不起作用！
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/frontend/nextjs.mdx]:
```suggestion
    版本 8.0.0 **无法**运作！
```
Maybe a better wording choice.

File: src/content/docs/zh-cn/start/frontend/nextjs.mdx [hunk 2]
```diff
@@ -1,103 +1,118 @@
...
+
+    <CommandTabs
+      npm="npm install --save-dev internal-ip@7.0.0"
+      yarn="yarn add -D internal-ip@7.0.0"
+      pnpm="pnpm add -D internal-ip@7.0.0"
+    />
+
+1.  ##### 更新金牛座配置
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/frontend/nextjs.mdx]:
```suggestion
1.  ##### 更新 Tauri 配置
```

File: src/content/docs/zh-cn/start/frontend/nextjs.mdx [hunk 3]
```diff
@@ -1,103 +1,118 @@
...
+  minHeadingLevel: 2
+  maxHeadingLevel: 5
 ---
 
-import { Tabs, TabItem } from '@astrojs/starlight/components';
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
 import CommandTabs from '@components/CommandTabs.astro';
 
-Next.js 是一个 React 框架。在 https://nextjs.org 获取更多有关 Next.js 的信息。本指南基于 Next.js 13.4.19。
+Next.js 是一个基于 React 的元框架。了解更多关于 Next.js 的信息，请访问 https://nextjs.org 。本指南适用于 Next.js 14.2.3 版本。
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/frontend/nextjs.mdx]:
```suggestion
Next.js 是一个基于 React 的元框架。要了解更多关于 Next.js 的信息，请访问 https://nextjs.org 。本指南适用于 Next.js 14.2.3 版本。
```
wording adjustment.

File: src/content/docs/zh-cn/start/frontend/nuxt.mdx [hunk 4]
```diff
@@ -1,102 +1,111 @@
...
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 5
 ---
 
-import { Tabs, TabItem } from '@astrojs/starlight/components';
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
 
-前往 https://nuxt.com 获取更多有关 Nuxt 的信息。本指南基于 Nuxt 3.7。
+Nuxt 是一个基于 Vue 的元框架。了解更多关于 Nuxt 的信息，请访问 https://nuxt.com 。本指南适用于 Nuxt 3.7版本。
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/frontend/nuxt.mdx]:
```suggestion
Nuxt 是一个基于 Vue 的元框架。要了解更多关于 Nuxt 的信息，请访问 https://nuxt.com 。本指南适用于 Nuxt 3.7版本。
```
wording adjustment.

File: src/content/docs/zh-cn/start/frontend/nextjs.mdx [hunk 5]
```diff
@@ -1,103 +1,118 @@
...
 
-## 检查清单
+## 清单
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/frontend/nextjs.mdx]:
```suggestion
## 检查清单
```
检查清单 should make more sense as a literal 清单 makes no sense.

File: src/content/docs/zh-cn/start/frontend/nuxt.mdx [hunk 6]
```diff
@@ -1,102 +1,111 @@
...
 
-## 检查清单
+## 清单
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/frontend/nuxt.mdx]:
```suggestion
## 检查清单
```
检查清单 should make more sense as a literal 清单 makes no sense.

File: src/content/docs/zh-cn/start/frontend/trunk.mdx [hunk 7]
```diff
@@ -1,11 +1,16 @@
...
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 5
 ---
 
-Trunk 是一个用于 Rust 的 WASM 网页应用打包器。 前往 https://trunkrs.dev 获取更多信息。本指南基于 Trunk 0.17.5。
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
 
-## 检查清单
+Trunk 是一个用于 Rust 的 WASM 网络应用程序打包工具。了解更多关于 Trunk 的信息，请访问 https://trunkrs.dev 。本指南适用于 Trunk 0.17.5 版本。
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/frontend/trunk.mdx]:
```suggestion
Trunk 是一个用于 Rust 的 WASM 网络应用程序打包工具。要了解更多关于 Trunk 的信息，请访问 https://trunkrs.dev 。本指南适用于 Trunk 0.17.5 版本。
```
Wording adjustment.

File: src/content/docs/zh-cn/start/frontend/trunk.mdx [hunk 8]
```diff
@@ -1,11 +1,16 @@
...
+
+## 清单
```
Comment (dklassic) [inline src/content/docs/zh-cn/start/frontend/trunk.mdx]:
```suggestion
## 检查清单
```
检查清单 should make more sense as a literal 清单 makes no sense.

**Review by dklassic — CHANGES_REQUESTED** [review]
Hi, thanks for contributing.

Your recent contributions seems to be derivative of machine assisted translation, and while I'm totally open to the idea of using MT in translating Tauri docs I'd like to ask you kindly to put more effort into validating the content less another obvious error such as `Tauri` being translated into `金牛座` made it through.

**Comment by GuoJikun** [comment]
> Hi, thanks for contributing.
> 
> Your recent contributions seems to be derivative of machine assisted translation, and while I'm totally open to the idea of using MT in translating Tauri docs I'd like to ask you kindly to put more effort into validating the content less another obvious error such as `Tauri` being translated into `金牛座` made it through.

Okay, I will pay attention.

**Review by dklassic — APPROVED** [review]

**Pull request merged**